tests/build-job-matrix.py

```diff
@@ -102,9 +102,9 @@ def update_config(parent: Dict[str, Any], child: Dict[str, Any]) -> Dict[str, An
 
     # TODO: design a configurable system for this
     process_count = {
-        "macos": {False: 0, True: 4}.get(conf["parallel"], conf["parallel"]),
-        "ubuntu": {False: 0, True: 4}.get(conf["parallel"], conf["parallel"]),
-        "windows": {False: 0, True: 2}.get(conf["parallel"], conf["parallel"]),
+        "macos": {False: 0, True: 4}.get(conf["parallel"], 4),
```

Comment:
It seems to me that if `parallel` is missing from `conf` - I don't see how this works either old or new?
If it's there, it should be either true or false, so I'm confused how this changes anything?